sdk/communication/azure-communication-rooms/README.md

```diff
@@ -53,71 +53,80 @@ public RoomsClient createRoomsClientWithConnectionString() {
 
 ## Key concepts
 
-There are four operations to interact with the Azure Communication Rooms Service.
+### Rooms
+- Create room
+- Update room
+- Get room
+- Delete room
+- List all rooms
+
+### Participants
+- Add or Update Participants
+- Remove Participants
```

Comment:
nit: keep the caps consistent
"Add or update participants."
"Remove participants"
"List all participants"